docs/guidelines.asciidoc

```diff
@@ -28,6 +27,8 @@ practices.
 * *Singular or plural.* Use singular and plural names properly to reflect the field content. For example, use `requests_per_sec` rather than `request_per_sec`.
 * *General to specific.* Organise the prefixes from general to specific to allow grouping fields into objects with a prefix like `host.*`.
 * *Avoid repetition.* Avoid stuttering of words. If part of the field name is already in the prefix, do not repeat it. Example: `host.host_ip` should be `host.ip`.
-* *Use prefixes.* Fields must be prefixed except for the base fields. For example all `host` fields are prefixed with `host.`. See `dot` notation in FAQ for more details.
+* *Use prefixes.* Fields must be prefixed except for the base fields. For example, all `host` fields are prefixed with `host.`. See `dot` notation in FAQ for more details.
```

Comment:
Here's the perfect place to address one of the small clarifications I wanted to address in time for GA.
```suggestion
* *Use prefixes.* Fields must be prefixed except for the base fields. For example, all `host` fields are prefixed with `host.`. See `dot` notation in FAQ for more details.
  * Note that the document structure should be nested JSON objects. If you use Beats or Logstash, the nesting of JSON objects is done for you automatically. If you're ingesting to Elasticsearch via the API, make sure your fields are nested objects, not strings containing dots.
```

Reply:
Trying to nest a bullet point under the "Use prefixes" guideline, not sure if this is the way in asciidoc.

This would address "Clarify on dots in field names" from the GA plan.

Reply:
I did something a little different. Take a look and let me know if you'd like to discuss.

Reply:
Yes, love the way you did it. 👍